tests/test_scripts.py

```diff
@@ -258,10 +258,33 @@ def test_filter_abund_3_fq_retained():
     seqs = set([r.sequence for r in screed.open(outfile)])
     assert len(seqs) == 2, seqs
     assert 'GGTTGACGGGGCTCAGGG' in seqs
+
     # check for 'quality' string.
-    seqs = set([r.quality for r in screed.open(outfile)])
-    assert len(seqs) == 2, seqs
-    assert '##################' in seqs
+    quals = set([r.quality for r in screed.open(outfile)])
+    assert len(quals) == 2, quals
+    assert '##################' in quals
+
+
+# make sure that FASTQ names are properly parsed, both formats.
+
+
+def test_filter_abund_4_fq_casava_18():
+    infile = utils.get_temp_filename('test.fq')
+    in_dir = os.path.dirname(infile)
+
+    shutil.copyfile(utils.get_test_data('test-abund-read-2.paired2.fq'),
```

Comment:
It would be nifty if Casava 1.8 formatted test sequence files had 'c18' in their name.